boilerplate/openshift/golang-osd-operator/README.md

```diff
@@ -47,4 +47,14 @@ run code coverage analysis per [this SOP](https://github.com/openshift/ops-sop/b
 - runs it against
 - a `golangci.yml` config.
 
+## Checks on generated code
+
+The convention embeds default checks to ensure generated code generation is current, committed, and unaltered.
+To trigger the check, you can use `make check-generate` provided your Makefile properly includes the boilerplate-generated include `boilerplate/generated-includes.mk`.
```

Comment:
Bleh, `check-generate` isn't really noun-verb. However, I still have a problem with using the noun form of `generate` (`generation`) because I feel it's easily confused with the "resource version" definition. I don't know that `make generate-check` is really any better, since those are both verbs -- even though the protocol seems to be putting "the thing that's being done" -- in this case the `check` -- last.

I throw up my hands. Ignore the above.

"A foolish consistency is the hobgoblin of little minds, adored by little statesmen and philosophers and divines."
-- Emerson

Reply:
✓ #59